.github/dependabot.yml

```diff
@@ -13,9 +13,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
-    # disable version updates for pip dependencies 
-    open-pull-requests-limit: 0    
```

Comment:
nit: Shall we still set a limit to prevent too many updates at the same time?

Reply:
I don't think there will be too many? I think we can do so when it becomes an issue.